src/HashBus.Projector.MostMentioned/MostMentionedProjection.cs

```diff
@@ -59,6 +60,8 @@ public void Handle(TweetAnalyzed message)
                     " mention to ".Gray(),
                     $" {message.Tweet.Track} ".DarkCyan().On(ConsoleColor.White));
             }
+
+            return Task.FromResult(0);

```

Comment:
given that we're targeting `4.6.1` we can use `Task.CompletedTask` as return value